admin_site/system/views.py

```diff
@@ -1874,3 +1895,50 @@ def get_context_data(self, **kwargs):
                 pc[pn] = pv
             pcs.append(pc)
         return pcs
+
+
+class ImageVersionsView(SiteMixin, SuperAdminOrThisSiteMixin, ListView):
+    """Image Versions are are sorted by major versions in the view
+    """
+
+    template_name = 'system/site_image_versions.html'
+    model = ImageVersion
+    context_object_name = 'image_versions'
+    selection_class = ImageVersion
+    class_display_name = 'image_version'
+
+    def get_context_data(self, **kwargs):
+        context = super(ImageVersionsView, self).get_context_data(**kwargs)
+
+        site_uid = self.kwargs.get('site_uid')
+        site_obj = Site.objects.get(uid=site_uid)
+        site_allowed = False
+        if site_obj.last_version:
+            site_allowed = True
+
+        context["site_allowed"] = site_allowed
+
+        versions = ImageVersion.objects.all()
```

Comment:
Er... is this right? Surely we should only show images whose release date is less than or equal to the customer's `last_version` date.